$NetBSD: patch-ah,v 1.18 2023/04/30 14:58:58 spz Exp $

don't install in site, we're installing in samples and then doing
CONF_FILES_PERMS from that

--- site/Makefile.orig	2016-11-27 14:03:42.000000000 +0000
+++ site/Makefile
@@ -75,7 +75,8 @@ REST		= \
 
 ALL		= $(REST)
 
-REST_INSTALLED	= \
+REST_INSTALLED	= 
+NOTUSED		= \
 	$D$(PATH_NEWSFEEDS) $D$(PATH_INNDHOSTS) \
 	$D$(PATH_NNRPDTRACK) $D$(PATH_NNTPPASS) \
 	$D$(PATH_CONFIG) $D$(PATH_MODERATORS) \
@@ -102,7 +103,8 @@ REST_INSTALLED	= \
 
 ALL_INSTALLED	= $(REST_INSTALLED)
 
-SPECIAL		= $D$(PATH_ACTIVE) $D$(PATH_ACTIVE_TIMES) \
+SPECIAL		=
+NOTUSED2	= $D$(PATH_ACTIVE) $D$(PATH_ACTIVE_TIMES) \
 		$D$(PATH_NEWSGROUPS) $D$(PATH_HISTORY)
 
 ##  Get new versions of everything from samples directory.
